src/apollo/client.ts

```diff
@@ -9,15 +9,74 @@ import {
 } from "@apollo/client";
 import { onError } from "@apollo/client/link/error";
 import { setContext } from "@apollo/client/link/context";
-import fetch from "cross-fetch";
+import jwtDecode from "jwt-decode";
 import { TypedTypePolicies } from "./apollo-helpers";
 import { REFRESH_TOKEN } from "./mutations";
 import { RefreshTokenMutation, RefreshTokenMutationVariables } from "./types";
 import { storage } from "../core/storage";
 
+type JWTToken = {
+  iat: number;
+  iss: string;
+  owner: string;
+  exp: number;
+  token: string;
+  email: string;
+  type: string;
+  user_id: string;
+  is_staff: boolean;
+};
+
 let client: ApolloClient<NormalizedCacheObject> | undefined;
 
-export const authLink = setContext((_, { headers }) => {
+const refreshToken = async (
```

Comment:
This should be part of the SDK `auth` methods

Reply:
done